src/main/kotlin/de/cyface/collector/handler/AuthorizationHandler.kt

```diff
@@ -63,159 +55,29 @@ class AuthorizationHandler(
             LOGGER.debug("Request headers: {}", headers)
 
             // Check authorization
-            val principal = UsernamePasswordCredentials(context.user().principal())
-            val username = principal.username
+            // FIXME: double check only authenticated users arrive here
+            // TODO: use OAuth2 roles to handle group permissions, but not required in collector
```

Comment:
**re FIXME:**
- Before we called in CollectorVerticel:
```
- jwtHandler = JWTAuthHandler.create(jwtAuth)
- then AuthorizationHandler(authProvider, ...) which is this handler here.
```
I.e. we first authenticated the user, and the in this `AuthenorizationHandler` here we called `authProvider.authenticate(principal)` again to get the `roles` of that user which were not provided in the previous auth result.

Now in `CollectorVerticle` (WIP) we register in `registerPreRequestHandler()`
- oauth2Handler to authenticate the users
- so there is no need to do a second authenticate request in preRequestHandlerAuthorizationhandler as we don't need to load any roles right now in the collector.


**re TODO:**
- As only the authenticated user can upload files in its own name, there is no need to load `accessible-users`.
- I only did this in `de.cyface.api` Library as this was also used by the data-provider where there was is the concept of "manager" users.